packages/portal/src/pages/search/index.vue

```diff
@@ -5,16 +5,38 @@
     >
       <template
         v-if="!!searchQuery"
-        #related
+        #related-galleries
       >
-        <client-only>
-          <RelatedSection
-            :query="searchQuery"
-            :overrides="relatedCollections"
-            data-qa="related section"
-            @fetched="handleRelatedSectionFetched"
-          />
-        </client-only>
+        <aside
+          v-masonry-tile
```

Comment:
This is applied even when using the list view. Is that a problem?

Reply:
I changed it so the directive is active only when not in list view (also for other masonry tiles).